README.md

```diff
@@ -297,10 +298,44 @@ jobs:
     - uses: actions/checkout@...
     - uses: deadsnakes/action@...
       with:
-          python-version: '3.13-dev'
+          python-version: '3.13.0b2'
           nogil: true
 ```
 
-You should replace the ellipses with versions for the actions and you can
-replace `3.13-dev` with a version like `3.13.0b1` once there is a build for
-that version in the `deadsnakes` PPA.
+You should replace the ellipses with versions for the actions. If there is a
+newer CPython 3.13 release available since this document was written or
+updated, use that version instead.
+
+The [cibuildwheel](https://cibuildwheel.pypa.io/en/stable/) project has support
+for building free-threaded wheels on all platforms. If your project releases
+nightly wheels, we suggest configuring cibuildwheel to build nightly
+free-threaded wheels.
+
+You will need to specify the following variables in the environment for the
+cibuildwheel action:
+
+```
+      - name: Build wheels
+        uses: pypa/cibuildwheel@...
+        env:
+          CIBW_PRERELEASE_PYTHONS: True
+          CIBW_FREE_THREADED_SUPPORT: True
+          CIBW_BUILD: cp313t-${{ matrix.buildplat }}
+          # TODO: remove along with installing build deps in
+          # cibw_before_build.sh when a released cython can build numpy
+          CIBW_BUILD_FRONTEND: "pip; args: --no-build-isolation"
+```
+
+As above, replace the ellipses with a `cibuildwheel` version.
+
+If your project depends on cython, you will need to install a Cython nightly
+wheel in the build, as the newest stable release of Cython cannot generate code
+that will compile under the free-threaded build. You likely do not need to
+disable pip build isolation if your project does not depend on cython.
```

Comment:
Maybe suggest installing nightly wheels for NumPy and SciPy as well, in case they're dependencies? Also, including the `pip` command to do so would probably be helpful.